articles/azure-monitor/app/worker-service.md

```diff
@@ -3,7 +3,7 @@ title: Application Insights for Worker Service apps (non-HTTP apps) | Microsoft
 description: Monitoring .NET Core/.NET Framework non-HTTP apps with Application Insights.
 services: application-insights
 documentationcenter: .net
-author: cithomas
+author: cijothomas
```

Comment:
why this change?

Reply:
This field needs to include the GitHub alias of the author. Otherwise, the tool (and PR reviewers) can't find the author.